sdk/textanalytics/azure-ai-textanalytics/src/test/resources/session-records/analyzeSentimentBatchTooManyDocuments.json

```diff
@@ -1,22 +1,22 @@
 {
   "networkCallRecords" : [ {
     "Method" : "POST",
-    "Uri" : "https://REDACTED.azure-api.net/text/analytics/v3.1-preview.2//sentiment",
+    "Uri" : "https://REDACTED.azure-api.net/text/analytics/v3.1-preview.2//sentiment?stringIndexType=Utf16CodeUnit",
```

Comment:
Is this test updated to show that the stringIndexType specification has no effect when used on a regular statement?
Or is it a required param and defaulted on client-side?

Reply:
It is manually defaulted on client-side.
Currently, we set the StringIndexType to Utf16CodeUnit internally, https://github.com/Azure/azure-sdk-for-java/pull/14784/files#diff-cebc2c113934f42da10998bc0ff20929R219. There are two other type: TextElements_v8 and UnicodeCodePoint. In the future, we might explore the StringIndexType to user.